packages/environment/develop.js

```diff
@@ -113,29 +113,25 @@ const Develop = {
     });
   },
 
-  connectOrStart: async function(options, ganacheOptions) {
+  connectOrStart: async function (options, ganacheOptions) {
     options.retry = false;
 
     const ipcNetwork = options.network || "develop";
 
-    let connectedAlready = false;
+    let started = false;
+    let disconnect;
 
     try {
-      const disconnect = await this.connect(options);
-      connectedAlready = true;
-      return {
-        started: false,
-        disconnect
-      };
+      disconnect = await this.connect(options);
     } catch (_error) {
       await this.start(ipcNetwork, ganacheOptions);
       options.retry = true;
-      const disconnect = await this.connect(options);
-      if (connectedAlready) return;
```

Comment:
Huh, there's no way this could ever be hit, is there?  And then it does something nonsensical too.  Yikes.  Glad to see that gone.